mitmproxy/tools/web/master.py

```diff
@@ -51,6 +52,7 @@ def __init__(self, opts: options.Options, with_termlog: bool = True):
         self.app = app.Application(self, self.options.web_debug)
         self.proxyserver: Proxyserver = self.addons.get("proxyserver")
         self.proxyserver.servers.changed.connect(self._sig_servers_changed)
+        self.filters_manager = FiltersManager(self.view)
```

Comment:
this is useless now, but ideally it will be helpful when we evaluate the filters on a single flow (add or update)